packages/docs/content/docs/code/page-navigation.mdx

```diff
@@ -32,6 +40,113 @@ const PageNavigation = () => {
 export default PageNavigation;
 ```
 
-The example above shows:
+## Navigation Buttons Implementation
+
+Create custom navigation controls with page jumping and input:
+
+```tsx
+import { usePdf, usePdfJump } from "@unriddle-ai/lector";
+import { useEffect, useState } from "react";
+
+const PageNavigationButtons = () => {
+  const pages = usePdf((state) => state.pdfDocumentProxy?.numPages);
+  const currentPage = usePdf((state) => state.currentPage);
+  const [pageNumber, setPageNumber] = useState<string | number>(currentPage);
+  const { jumpToPage } = usePdfJump();
+
+  const handlePreviousPage = () => {
+    if (currentPage > 1) {
+      jumpToPage(currentPage - 1, { behavior: "auto" });
+    }
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < pages) {
+      jumpToPage(currentPage + 1, { behavior: "auto" });
+    }
+  };
+
+  useEffect(() => {
+    setPageNumber(currentPage);
+  }, [currentPage]);
+
+  return (
+    <div className="absolute bottom-8 left-1/2 transform -translate-x-1/2 flex items-center gap-3 bg-white/90 backdrop-blur-sm rounded-full shadow-lg px-4 py-2.5 border border-gray-200">
+      <button
+        onClick={handlePreviousPage}
+        disabled={currentPage <= 1}
+        className="p-1.5 hover:bg-gray-100 rounded-full transition-colors disabled:opacity-40"
```

Comment:
don't use button from here, use the shacdn buttons from components

also the docs have access to shacdn UI , so don't use tailwind colors, just always the shacdn theme

https://ui.shadcn.com/docs/theming